composer.json

```diff
@@ -26,6 +26,7 @@
         "twig/twig": "~1.12 || ~2.0"
     },
     "require-dev": {
+        "bossa/phpspec2-expect": "^3.0",
```

Comment:
We already have `Webmozart\Assert` dependency so if you prefer `bossa/phpspec2-expect` it's totally fine for me but you need to refactor specs written with Webmozart then :)